File: Results/DER/Translation/translation_output/w_test/gpt-3.5/codenet/java/python/p03219.py
```python
def upper_bound(a, val, l=0, r=None):
    if r is None:
        r = len(a)
    if r - l == 1:
        if a[l] > val:
            return l
        return r
    mid = (l + r) // 2
    if a[mid] > val:
        return upper_bound(a, val, l, mid)
    else:
        return upper_bound(a, val, mid, r)

def lower_bound(a, val, l=0, r=None):
    if r is None:
        r = len(a)
    if r - l == 1:
        if a[l] < val:
            return r
        return l
    mid = (l + r) // 2
    if a[mid] < val:
        return lower_bound(a, val, mid, r)
    else:
        return lower_bound(a, val, l, mid)

def put(object):
    print(object)

x, y = map(int, input().split())
put(x + y // 2)
```

File: Results/DER/Translation/translation_output/misleading_test/granite/codenet/python/java/p03219.java
System:
You are an expert java programmer and assistant

Question:
Translate the following code from python to java and enclose your solution inside ```java```.
A sample test case is provided below:

Test input:
81 58

Expected output:
127

```
ab, bc = map(int, input().split())

fare = int(ab + bc/2)

print(fare)


```


Answer:
```java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int ab = scanner.nextInt();
        int bc = scanner.nextInt();
        int fare = ab + bc / 2;
        System.out.println(fare);
    }
}
```

Test input:
```
81 58
```

Expected output:
```
127
```<|endoftext|>
